packages/calico-mesos.spec

```diff
@@ -31,13 +32,17 @@ mkdir -p -m 0755 %{buildroot}/calico
 install -m 0755 %{SOURCE0} %{buildroot}/calico
 install -m 0755 %{SOURCE2} %{buildroot}/calico
 
+mkdir -p -m 0755 %{buildroot}%{_sysconfdir}/mesos-slave
+install -m 0755 %{SOURCE4} %{buildroot}%{_sysconfdir}/mesos-slave/
+
 ############################################
 %files
 
 /calico/calico_mesos
 /calico/modules.json
-/%{_unitdir}/%{name}.service
-/%{_bindir}/calicoctl
+%{_unitdir}/%{name}.service
+%{_bindir}/calicoctl
+%{_sysconfdir}/mesos-slave/modules
```

Comment:
just trying to follow what's in this change:

Does Mesos slave look in `../mesos-slave/modules/` to load the `modules.json` or something?  Or am I missing how this gets the agent to load the modules now?